### PR TITLE
Refactor/unittest blender transition window

### PR DESCRIPTION
**Pull request by martiniil**
* add more test cases
* remove redundant test cases and tests of methods not owned by this unit

Refactoring of the testdata_loader is postponed to later pull requests (or rebase).

### CODE REVIEW

File: pilz_trajectory_generation/test/unittest_trajectory_blender_transition_window.cpp [hunk 1]
```diff
@@ -525,6 +568,200 @@ TEST_P(TrajectoryBlenderTransitionWindowTest, testLINLINBlending)
...
+                                          blend_res,
+                                          planner_limits_,
+                                          joint_velocity_tolerance_,
+                                          joint_acceleration_tolerance_,
+                                          cartesian_velocity_tolerance_,
+                                          cartesian_angular_velocity_tolerance_));
+}
+
+/**
+ * @brief  Tests the blending of two cartesian trajectories generated by adding sine curves to linear trajectories
```
Comment (agutenkunst) [inline pilz_trajectory_generation/test/unittest_trajectory_blender_transition_window.cpp]: Could you explain a bit further the reasoning behind this?

File: pilz_trajectory_generation/test/unittest_trajectory_blender_transition_window.cpp [hunk 2]
```diff
@@ -525,6 +568,200 @@ TEST_P(TrajectoryBlenderTransitionWindowTest, testLINLINBlending)
 
 }
 
+/**
+ * @brief  Tests the blending of two cartesian linear trajectories with a non-trivial overlap using robot model
```
Comment (agutenkunst) [inline pilz_trajectory_generation/test/unittest_trajectory_blender_transition_window.cpp]: "non-trivial"?

**Review by agutenkunst — CHANGES_REQUESTED** [review]
Just some minor changes.

File: pilz_trajectory_generation/test/unittest_trajectory_blender_transition_window.cpp (hunk 1 above)
Comment (martiniil) [reply pilz_trajectory_generation/test/unittest_trajectory_blender_transition_window.cpp]: The whole idea is to blend trajectories, that differ from a straight line. I changed this line. The procedure of adding the sine is explained further down in the comment anyways.

File: pilz_trajectory_generation/test/unittest_trajectory_blender_transition_window.cpp (hunk 2 above)
Comment (martiniil) [reply pilz_trajectory_generation/test/unittest_trajectory_blender_transition_window.cpp]: Thanks. I did reformulate it.

**Comment by agutenkunst** [comment]
Closing for https://github.com/PilzDE/pilz_industrial_motion/pull/79

**Pull request closed without merging**